components/core/header/nav-bar/NavBar.vue

```diff
@@ -3,12 +3,19 @@
         <locale-link to="/about" :class="getPageClassesByPath('about', true)">
             {{ $t('about') }}
         </locale-link>
+        <locale-link
+            to="/sponsor"
+            :class="getPageClassesByPath('sponsor', true)"
+        >
+            {{ $t('sponsor') }}
+        </locale-link>
```

Comment:
We do not have nav bar item for prospectus in [2020 website](https://github.com/pycontw/pycon.tw/blob/master/src/templates/pycontw-2020/_includes/menu.html#L46_L50). I think it's reasonable since this page is only for sponsors  and I just followed it